run_pacta_data_preparation.R

```diff
@@ -64,6 +64,24 @@ scenario_geographies_list <- config$scenario_geographies_list
 global_aggregate_scenario_sources_list <- config$global_aggregate_scenario_sources_list
 global_aggregate_sector_list <- config$global_aggregate_sector_list
 
+# create timestamped output directory
+system_timestamp <- format(
+  Sys.time(),
+  format = "%Y%m%dT%H%M%SZ",
+  tz = "UTC"
+)
+data_prep_outputs_path <- file.path(
+  data_prep_outputs_path,
+  paste(pacta_financial_timestamp, system_timestamp, sep = "_")
```

Comment:
should the directory include any suffix indicating what it is, e.g. `*_dataprep_outputs`?

Reply:
I think having the inputs archive called out with an `_inputs` suffix (in #159) differentiates these, combined with the fact that our Azure storage puts these all in `workflow-data-prep-outputs`. If you want more differentiation though, I'm okay with that.

Reply:
nope that's fine... just checking that it wasn't an oversight